beebop/versions.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from beebop import __version__
+
+
+def get_version(components):
+    """
+    report version numbers for all components requested
+
+    input : list with components like beebop (also poppunk & ska in the future)
+
+    output : json according to 'version' schema
+    """
+
+    arr = []
+    for x in components:
+        entry = {
+            "name": x,
+            "version": __version__  # needs to be adapted if including other components
+        }
+        arr.append(entry)
+
+    result = json.dumps(arr)
```

Comment:
There's a big question about who does the serialisation and where. This function will be much easier to test if it returns the data and if the flask part takes care of the serialisation

Reply:
OK, looks like what flask will do is:

> If it’s a dict, a response object is created using jsonify.

What we do in hintr and other APIs is that the response always looks like:


```json
{
    "status": "success",
    "errors": [
    ],
    "data": {
        "hash": "12345",
        "type": "pjnz",
        "data": {
            "country": "Botswana",
            "iso3": "BWA"
        },
        "filename": "Botswana2018.PJNZ",
        "filters": null
    }
}
```

with the `status`, `errors` and `data` keys being common (see https://github.com/reside-ic/porcelain/blob/master/inst/schema/response-success.schema.json and https://github.com/reside-ic/porcelain/blob/master/inst/schema/response-failure.schema.json for the schemas).  We might adopt this here, then we can ignore the details of serialisation later and let flask to it for us.